euctr/templates/_base.html

```diff
@@ -28,18 +28,7 @@
       {% include "_load_js.html" %}
     {% endif %}
 
-    {% if GOOGLE_TRACKING_ID %}
```

Comment:
should we remove the other references to GOOGLE_TRACKING_ID while we're here?

Reply:
Yes, good point - I'll add a commit now.

Reply:
@madwort updated, I assume I'm ok to remove the `euctr/frontend/context_processors.py` file as it only had references to `google_tracking_id` in it...?

Reply:
looks ok to me.